python-avd/pyavd/_eos_designs/fabric_documentation_facts/__init__.py

```diff
@@ -286,7 +286,7 @@ def all_connected_endpoints(self) -> dict[str, list]:
                         "peer_type": get(ethernet_interface, "metadata.peer_type"),
                         "peer_interface": get(ethernet_interface, "metadata.peer_interface", default="-"),
                         "fabric_switch": hostname,
-                        "fabric_port": ethernet_interface["name"],
+                        "fabric_port": get(port_channel_interface, "name") or ethernet_interface["name"],
```

Comment:
I think you may need to get more stuff adjusted
lke the description will be different, the port-channel may be shutdown or not even if one of its members is shutdown ... 

Can you take a second look at this?